src/frontend/src/handler/create_mv.rs

```diff
@@ -81,13 +77,13 @@ pub fn gen_create_mv_plan(
         }
     }
 
-    let mut plan_root = Planner::new(context).plan_query(bound)?;
+    let mut plan_root = Planner::new(context.clone()).plan_query(bound)?;
     plan_root.set_required_dist(RequiredDist::Any);
     let materialize = plan_root.gen_create_mv_plan(table_name)?;
     let mut table = materialize.table().to_prost(schema_id, database_id);
     let plan: PlanRef = materialize.into();
     table.owner = session.user_id();
-    table.properties = properties;
+    table.properties = context.inner().with_properties.clone();
```

Comment:
maybe we can add the properties in `StreamMaterialize::create`

Reply:
fix it cc @st1page